vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import tsconfigPaths from 'vite-tsconfig-paths';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  plugins: [tsconfigPaths()],
+  test: {
```

Comment:
couple things here in terms of coverage. I used all vitest defaults, including c8 for coverage instead of istambul, and I think it added 'text' to the reporters listed in the jestconfig.